common/ts/publish-task.ts

```diff
@@ -79,7 +79,12 @@ export async function getReportForTask(
   } catch (e) {
     if (e instanceof TimeOutReachedError) {
       tl.warning(
-        `Task '${taskReport.ceTaskId}' takes too long to complete. Stopping after ${timeoutSec}s of polling. No quality gate will be displayed on build result.`
+        "Task '" +
```

Comment:
I'm wondering why this change? It's the same as before but harder to read?

Reply:
For some reason few issues have been raised on next regarding line length, that's why I needed to wrap it (I also enforced it in the eslintrc file)

Reply:
I would typically close as won't fix such an error, because in this case, fixing the error makes the code less maintainable than before...

Reply:
Or you could eventually fix it in a cleaner way, maybe like:
```ts
      tl.warning(
        `Task '${taskReport.ceTaskId}' takes too long to complete. `
        `Stopping after ${timeoutSec}s of polling. No quality gate will be displayed on build result.`
```

Reply:
Ok that makes sense and i do agree thanks !